packages/gatsby-source-filesystem/src/gatsby-node.js

```diff
@@ -85,9 +91,32 @@ const createFSMachine = (
             },
             READY: {
               on: {
-                CHOKIDAR_ADD: { actions: `createAndProcessNode` },
-                CHOKIDAR_CHANGE: { actions: `createAndProcessNode` },
-                CHOKIDAR_UNLINK: { actions: `deletePathNode` },
+                CHOKIDAR_ADD: {
+                  actions: [
+                    `createAndProcessNode`,
+                    log(
+                      (_, { pathType, path }) => `added ${pathType} at ${path}`
+                    ),
+                  ],
+                },
+                CHOKIDAR_CHANGE: {
+                  actions: [
+                    `createAndProcessNode`,
+                    log(
+                      (_, { pathType, path }) =>
+                        `changed ${pathType} at ${path}`
+                    ),
+                  ],
+                },
+                CHOKIDAR_UNLINK: {
+                  actions: [
+                    `deletePathNode`,
+                    log(
+                      (_, { pathType, path }) =>
+                        `${pathType} deleted at ${path}`
```

Comment:
why not same style as others `{action} ${pathType} at ${path}`
```suggestion
                        deleted `${pathType} at ${path}`
```

Reply:
This just moved what we currently log, but I do think this suggestions makes sense (nicer to read)

Reply:
Great suggestion @muescha

Reply:
Ehhh, no. You should probably test this before merging. I promise you it does not what you think it does :joy:

Reply:
@pvdz did you mean my wrong backtick? or other problems?

Reply:
.

Reply:
Yeah the `deleted` was put before the backtick, causing it to become a tagged template. Syntactically valid, but runtime error because `deleted` is not a variable :)